Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -0,0 +1,57 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup>
+    <AssemblyName>Expecto.netcore</AssemblyName>
+    <Version>1.0.0-preview3</Version>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DebugType>pdbonly</DebugType>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
it's useles, is already defined in msbuild for Debug => `DEBUG`, Release => `RELEASE`